assets/openshift-apiserver/openshift-apiserver-deployment.yaml

```diff
@@ -151,6 +151,16 @@ spec:
           timeoutSeconds: 1
           failureThreshold: 3
           successThreshold: 1
+        startupProbe:
```

Comment:
We don't do need a startup probe for this.

Reply:
Ok, I saw this added upstream in the same commit as the shutdown-duration-delay flag, so I included it here as well. I removed it in the latest commit.